examples/README.md

```diff
@@ -58,4 +58,22 @@ Using tomography to analyse the effect of noise channels when iterating the Enta
 [Forest_portability_example](https://github.com/CQCL/pytket/blob/master/examples/Forest_portability_example.ipynb) -
 Examples illustrating portability between different backends.
 
-[qiskit_integration](https://github.com/CQCL/pytket/blob/master/examples/qiskit_integration.ipynb) - Wrapping a pytket backend as a qiskit backend.
+## Notes for developers
+
+The sources for all these notebooks are the Python scripts in the `python`
+directory. The notebook files are generated from them with the `gen-nb` script
+(which requires `p2j`). Do not edit the notebooks directly; instead edit the
+Python scripts and regenerate the notebooks by running `gen-nb`.
+
+The `check-examples` script (which is run regularly on the CI and can also be
+run locally) first checks that the notebooks match what is generated by `p2j`
+from the Python scripts; and then attempts to run all the scripts. For those
+scripts that require credentials, it first tries to load them, and skips the
+test if it fails. For those that require containers to be running, it first
+tries to start them, and skips the test if it fails.
+
+To add a new script, you can either create the Python file and convert it using
+`gen-nb` or create a notebook file, convert it to Python using `p2j -r`, and
+then run `gen-nb` to convert back. In either case, please check that the
+formatting in the notebook is satisfactory. Then add the name of the new
+notebook to `maintained-notebooks.txt`.
```

Comment:
You should mention that the name of the notebook needs to be added to the workflow in line 13-16 or 28 if login credentials are needed.